README.md

```diff
@@ -1,19 +1,42 @@
-# rtwg_documentation
+# ROS 2 Real-Time Working Group Documentation
 
 [![Documentation Status](https://readthedocs.org/projects/real-time-working-group/badge/?version=latest)](https://real-time-working-group.readthedocs.io/en/latest/?badge=latest)
 
-## On-line documentation link
+This repository contains the sources for the ROS 2 real-time working group documentation that is hosted at https://real-time-working-group.readthedocs.io.
+The sources from this repository are built and uploaded to the site manually.
 
-https://real-time-working-group.readthedocs.io/en/latest/
+## Contributing to the documentation
 
-## Build instructions
+Contributions to this site are most welcome.
+Please be sure to read the below sections carefully before contributing.
 
+The site is built using [Sphinx](https://www.sphinx-doc.org/)
+
+### Branch structure
+
+This repository is setup with one branch per ROS 2 distribution to handle differences between the distributions.
+If a change is common to all ROS 2 distributions, it should be made to the `rolling` branch (and then will be backported as appropriate).
+If a change is specific to a particular ROS 2 distribution, it should be made to the respective branch.
```

Comment:
does the section and the lines above really apply? We're only using `master` for now.

Reply:
@christophebedard - this is what we have discussed last week.  We need to change the branch name ```master``` to ```rolling``` or ```foxy```